en/faq/deployment-aws-s3-cloudfront.md

```diff
@@ -1,57 +1,84 @@
 ---
-title: "AWS: S3+Cloudfront Deployment (En)"
-description: Static Hosting on AWS with S3 and Cloudfront
+title: Déploiement S3+Cloudfront de AWS
+description: Hébergement statique sur AWS avec S3 et Cloudfront
 ---
 
-# How to Deploy on AWS w/ S3 and Cloudfront
+# Comment déployer sur AWS avec S3 et Cloudfront
 
-<p style="width: 294px;position: fixed; top : 64px; right: 4px;" class="Alert Alert--orange"><strong>⚠Cette page est actuellement en cours de traduction française. Vous pouvez repasser plus tard ou <a href="https://github.com/vuejs-fr/nuxt" target="_blank">participer à la traduction</a> de celle-ci dès maintenant !</strong></p><p>AWS is Amazon Web Services.</p>
-S3 is their static storage which can be configured for Static Site Hosting.
-Cloudfront is their CDN (content delivery network)
+AWS est un service web d'Amazon.
+S3 est leur stockage statique qui peut être configuré pour de l'hébergement de site statique.
+Cloudfront est leur CDN (content delivery network)
 
-Hosting a Nuxt "Universal App" on AWS w/ S3 + Cloudfront is powerful and cheap.
+Héberger une "application universelle" Nuxt sur AWS avec S3 et Cloudfront est puissant et pas cher.
 
-> AWS is a death by 1000 paper cuts.  If we missed a step, please submit a PR to update this document.
+> AWS est à se tirer les cheuveux. Si nous avons oublié une étape, merci de soumettre proposition de mise à jour (PR) pour mettre à jour ce document.
 
-## Overview
+## Tour rapide
 
-We will use the [Gulp](https://gulpjs.com/) build system for this.  `Gulp` is a mature build system with handy utilities we can use.
+Nous allons héberger à pas cher avec quelques services AWS. Brièvement :
```

Comment:
???

Reply:
phrase en anglais :
"We'll host super cheap with some AWS services.  Briefly:"

si vous avez une idée pour la traduction ?

Reply:
Pour moi le mieux serait d'enlever la phrase en français et de faire une PR sur le repo anglais pour l'enlever également.